main.py

```
@@ -98,9 +105,14 @@ async def on_ready():
             Translator: BVI.BVWebTranslator = None
             for i in range(len(rows)):
-                msg = await bot.get_channel(rows[i][2]).fetch_message(rows[i][1])
-                view = await pollLink(None, rows[i][3])
-                await msg.edit(view=view)
+                try:
+                    print(rows[i][2])
+                    print(rows[i][1])
+                    msg = await bot.get_channel(rows[i][2]).fetch_message(rows[i][1])
+                    view = await pollLink(None, rows[i][3])
+                    await msg.edit(view=view)
+                except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)